tasks/post-deploy/06-wp58.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+if [ "$APP_ENV" = "development" ]; then
+  wp core update --version=5.8.1
```

Comment:
Can't we specify the version in the image build instead? This should work but it's not ideal to have it build and deploy an image of an older version and then at runtime overwrite the files.

Reply:
We don't have an easy of doing that unfortunately, because we use the same image for all environments.

Reply:
Doesn't this check work? It seems like it should :thinking: https://github.com/greenpeace/planet4-docker/blob/a7b5df925fe688e118095144f8c6b66695077e14/src/planet-4-151612/wordpress/etc/my_init.d/20_install_wordpress.sh#L246

Reply:
Looks it's possible after all, since the last change we did on builder. So I'm closing this in favor of https://github.com/greenpeace/planet4-docker/pull/85. Thanks for the push!

`5.8` is already deployed on the Gutenberg instance.